Upstream-Status: Inappropriate [legacy version]

This patch was imported from the Fedora Core 8 coreutils-6.9-9 package.

The package is stated as being Licensed as GPLv2+.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

--- coreutils-5.2.1/src/who.c.overflow	2005-05-25 09:59:06.000000000 +0100
+++ coreutils-5.2.1/src/who.c	2005-05-25 10:00:31.000000000 +0100
@@ -75,7 +75,7 @@
 # define NEW_TIME 0
 #endif
 
-#define IDLESTR_LEN 6
+#define IDLESTR_LEN 10
 
 #if HAVE_STRUCT_XTMP_UT_PID
 # define PIDSTR_DECL_AND_INIT(Var, Utmp_ent) \
